components/app/PlaceItem.jsx

```diff
@@ -57,26 +58,28 @@ const Tag = ({ type, ...props }) => (
   </Box>
 )
 
-const PlaceInfo = ({ name, category, tags }) => (
+const PlaceInfo = ({ name, category, tags, collection, delivery }) => (
   <Box width="100%">
     <Row width="100%" justifyContent="space-between" mt={2} mb={1}>
       <CategoryName>
         {(category || '').toUpperCase()}
       </CategoryName>
       <CollectionMethod>
-        {(category || '').toUpperCase()}
+        {(delivery === "TRUE" ? "Delivery" : '')}
+        {(collection === "TRUE" && delivery === "TRUE" ? " & Collection" : collection === "TRUE" ? "Collection" : '')}
       </CollectionMethod>
     </Row>
     <PlaceName mb={2}>
-      {name}
+    <Link to={`/${_.kebabCase(name)}`} style={{ textDecoration: 'none' }}>{name}</Link>
     </PlaceName>
-    {tags && (
+    <TagText> {tags && (
```

Comment:
This `<TagText>` will break native platforms, e.g. Figma, etc. There'll be `<div>`s inside of a `span`/`p` element